report.php

```diff
@@ -27,13 +27,15 @@
 
 $id = required_param('id', PARAM_INT);   // Course module.
 $sortby = optional_param('sortby', 'dateasc', PARAM_ALPHA);
+$selecteduser = optional_param('selecteduser', 0, PARAM_INT);
+
 if (!in_array($sortby, [
     'dateasc',
     'datedesc',
     'firstnameasc',
     'firstnamedesc',
     'lastnameasc',
-    'lastnamedesc',
```

Comment:
Don't remove the comma.
It's a Moodle coding style thing for multi line written arrays, see https://moodledev.io/general/development/policies/codingstyle#array-syntax